source/nanoFramework.Json/packages.config

```diff
@@ -1,8 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="nanoFramework.CoreLibrary" version="1.7.2-preview.7" targetFramework="netnanoframework10" />
+  <package id="nanoFramework.Runtime.Native" version="1.2.1-preview.2" targetFramework="netnanoframework10" />
   <package id="nanoFramework.System.Collections" version="1.0.1-preview.6" targetFramework="netnanoframework10" />
   <package id="nanoFramework.System.Text" version="1.0.0-preview.16" targetFramework="netnanoframework10" />
   <package id="nanoFramework.Windows.Storage.Streams" version="1.8.2-preview.5" targetFramework="netnanoframework10" />
-  <package id="Nerdbank.GitVersioning" version="3.1.91" targetFramework="netnanoframework10" developmentDependency="true" />
+  <package id="Nerdbank.GitVersioning" version="3.2.5-beta" targetFramework="netnanoframework10" developmentDependency="true" />
```

Comment:
Please dont update this nuget, we need the stable one for the release pipeline

Reply:
OK